src/screens/CreatePost/elements/CreatePollContainer/CreatePollContainer.ios.js

```diff
@@ -21,10 +20,8 @@ function CreatePollContainer({
   selectedtime = {day: 1, hour: 0, minute: 0},
   ontimechanged = () => {},
   polls,
-  expiredobject = {day: 7, hour: 24},
-  isAnonimity = false
+  expiredobject = {day: 7, hour: 24}
 }) {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of `isAnonimity` and the dynamic color hook `useDynamicColors(isAnonimity)` might affect the color scheme of your application, especially if you were using it to toggle between different color modes based on the anonymity status. If this is not intentional, consider reintroducing these lines.
```diff
-  expiredobject = {day: 7, hour: 24}
+  expiredobject = {day: 7, hour: 24},
+  isAnonimity = false
 }) {
+ const dynamicColor = useDynamicColors(isAnonimity);
   const arrayContentToString = (arr) => {
     const newArray = arr.reduce((acc, current) => {
       acc.push(`${current}`);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->